pkg/runtime/reconciler.go

```diff
@@ -1064,6 +1042,50 @@ func (r *resourceReconciler) getEndpointURL(
 	return r.cfg.EndpointURL
 }
 
+// getResyncPeriod returns the duration a reconciler should use to drift remediate custom
+// resources. It attempts to retrieve the duration from the following sources, in this order:
+// 1. A resource-specific drift remediation period specified in the drift remediation
+//    configuration JSON.
+// 2. A resource-specific requeue on success period specified by the resource manager factory.
```

Comment:
Perhaps indicate that the resource manager factory is controller-specific, and thus this period is to be specified by controller authors.